examples/Example.SmartContract.SampleRoyaltyNEP11Token/SampleRoyaltyNEP11Token.cs

```diff
@@ -0,0 +1,202 @@
+using System;
+using System.ComponentModel;
+using System.Numerics;
+
+namespace NonDivisibleNEP11
+{
+    /// <inheritdoc />
+    [DisplayName("SampleRoyaltyNEP11Token")]
+    [ContractAuthor("core-dev", "dev@neo.org")]
+    [ContractVersion("0.0.1")]
+    [ContractDescription("A sample of NEP-11 Royalty Feature")]
+    [ContractSourceCode("https://github.com/neo-project/neo-devpack-dotnet/tree/master/examples/")]
+    [ContractPermission(Permission.WildCard, Method.WildCard)]
+    [SupportedStandards(NepStandard.Nep11)]
+    public class SampleRoyaltyNEP11Token : Nep11Token<Nep11TokenState>
+    {
+        #region Owner
+
+        private const byte PrefixOwner = 0xff;
+
+        private static readonly UInt160 InitialOwner = "NUuJw4C4XJFzxAvSZnFTfsNoWZytmQKXQP";
+
+        [Safe]
+        public static UInt160 GetOwner()
+        {
+            var currentOwner = Storage.Get(new[] { PrefixOwner });
+
+            if (currentOwner == null)
+                return InitialOwner;
+
+            return (UInt160)currentOwner;
+        }
+
+        private static bool IsOwner() => Runtime.CheckWitness(GetOwner());
+
+        public delegate void OnSetOwnerDelegate(UInt160 newOwner);
+
+        [DisplayName("SetOwner")]
+        public static event OnSetOwnerDelegate OnSetOwner;
+
+        public static void SetOwner(UInt160? newOwner)
+        {
+            if (IsOwner() == false)
+                throw new InvalidOperationException("No Authorization!");
+            if (newOwner != null && newOwner.IsValid && !newOwner.IsZero)
+            {
+                Storage.Put(new[] { PrefixOwner }, newOwner);
+                OnSetOwner(newOwner);
+            }
+        }
+
+        #endregion
+
+        #region Minter
+
+        private const byte PrefixMinter = 0xfd;
+
+        private const byte PrefixCounter = 0xee;
+
+        private static readonly UInt160 InitialMinter = "NUuJw4C4XJFzxAvSZnFTfsNoWZytmQKXQP";
+
+        [Safe]
+        public static UInt160 GetMinter()
+        {
+            var currentMinter = Storage.Get(new[] { PrefixMinter });
+
+            if (currentMinter == null)
+                return InitialMinter;
+
+            return (UInt160)currentMinter;
+        }
+
+        private static bool IsMinter() => Runtime.CheckWitness(GetMinter());
+
+        public delegate void OnSetMinterDelegate(UInt160 newMinter);
+
+        [DisplayName("SetMinter")]
+        public static event OnSetMinterDelegate OnSetMinter;
+
+        public static void SetMinter(UInt160? newMinter)
+        {
+            ExecutionEngine.Assert(IsOwner(), "No Authorization!");
+            if (newMinter != null && newMinter.IsValid)
+            {
+                Storage.Put(new[] { PrefixMinter }, newMinter);
+                OnSetMinter(newMinter);
+            }
+        }
+
+        public static void Mint(UInt160 to)
+        {
+            ExecutionEngine.Assert(IsOwner() || IsMinter(), "No Authorization!");
```

Comment:
```suggestion
            if (IsOwner() == false || IsMinter() == false)
                throw new InvalidOperationException("No Authorization!");
```
You had `IsOwener() == true || IsMinter() == true` don't process.